Tests/SwiftParserTest/translated/IfconfigExprTests.swift

```diff
@@ -312,12 +312,12 @@ final class IfconfigExprTests: XCTestCase {
   func testIfconfigExpr22() {
     assertParse(
       """
-      #if canImport(A, unknown: 2.2)
+      #if canImport(A, 1️⃣unknown: 2.2)
         let a = 1
       #endif
       """,
       diagnostics: [
-        // TODO: Old parser expected error on line 1: 2nd parameter of canImport should be labeled as _version or _underlyingVersion
+        DiagnosticSpec(message: "2nd parameter of canImport should be labeled as _version or _underlyingVersion")
```

Comment:
Is there fix it's for those labels?

Reply:
I think we cannot infer whether the user miss a `_version` or `_underlyingVersion` here, so I prefer not to add a fixit here.